src/Compiler/TaskModel.cpp

```diff
@@ -89,7 +89,7 @@ QVariant TaskModel::data(const QModelIndex &index, int role) const {
         case TaskStatus::Fail:
           return QIcon(":/failed.png");
         case TaskStatus::InProgress:
-          return QIcon(":/loading.png");
+          return QString(":/loading.gif");
```

Comment:
it doesn't make any sense. What is just return bool, e.g. true? or just 0 or 1 ?

Reply:
@TarasSydor ?

Reply:
fixed